### ISSUE TITLE
How to unmap default keybinds

### ISSUE THREAD

**Issue by oniatsu** [inkdropapp/inkdrop-vim#19]
Is there any way of unmapping dafault keybinds?

I want to fix confliction of default keybinds.

For example, a substitution command `s` works so late, because there are some 2 stroke keybinds starting from `s`. 
https://github.com/inkdropapp/inkdrop-vim/blob/master/keymaps/vim.json#L130
https://github.com/inkdropapp/inkdrop-vim/blob/master/keymaps/vim.json#L5-L6

The delay makes `s` command unuseful.

If a developping the unmapping way is difficult, I think these `s x` keybinds should be removed from default.
https://github.com/inkdropapp/inkdrop-vim/blob/master/keymaps/vim.json#L5-L6

**Comment by craftzdog** [comment]
You can set falsy value to the keystrokes which you wish to unbind in your `keymap.cson` like so:

```cson
".CodeMirror.vim-mode.normal-mode textarea":
  "s h": false
  "s k": false
```

**Comment by oniatsu** [comment]
Thanks! But the problem is not solved.

For example, I set some preferences on `keyconfig.cson` to make every keystrokes starting with `s` invalid.
```cson
".CodeMirror.vim-mode.normal-mode textarea":
  "s h": false
  "s k": false
".sidebar-menu":
  "s l": false
".sidebar-book-filter-menu":
  "s l": false
".note-list-bar":
  "s l": false
  "s h": false
".mde-preview":
  "s h": false
  "s k": false
```

I typed `s h` on a main textarea on vim normal mode, then nothing happened.
An expected action is replacing a string at cursor to `h`, because `s` means substitution action.

Probably, that is because the `s h` keybind is still valid.

**Comment by craftzdog** [comment]
Will reply next week

**Comment by craftzdog** [comment]
That makes sense.
So `s h` is bound as 'do nothing' and not actually canceling the keybinding.
Well, since those keybindings mess the existing default vim behavior, I will remove them.

**Comment by craftzdog** [comment]
Landed in v2.1.2🎉

**Comment by oniatsu** [comment]
That works well! Thank you.
